libfreerdp/codec/test/TestFreeRDPCodecProgressive.c

```diff
@@ -1185,10 +1185,19 @@ static WINPR_NORETURN(void usage(const char* name))
 
 static void print_codec_stats(const char* name, UINT64 timeNS)
 {
-	const double dectimems = timeNS / 1000000.0;
+	const double dectimems = (double)timeNS / 1000000.0;
```

Comment:
On one side, we have here `dectimems`, on the other one `timeNS`. Either way, whether we use snake case or camel case, this should be unique here, e.g. either `dec_time_ms` + `time_ns` or `decTimeMS` + `timeNS`

Also: I counting zeros takes some time to verify the number. Instead of `(double)timeNS / 1000000.0;`, I think `(double)timeNS / 1000.0 / 1000.0;` is much clearer (and is as a number faster verified)

Reply:
done